tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -3910,7 +4124,67 @@ public function bar() {
     public function bar() { echo 1; }
   });
 }',
-                self::$configurationOopPositionSameLine + self::$configurationCtrlStructPositionNextLine,
+                self::$configurationOopPositionSameLine + self::$configurationAnonymousPositionNextLine,
+            ],
+            [
+                '<?php
+if (1) {
+    $message = (new class() extends Foo
+    {
+        public function bar() {
+            echo 1;
+        }
+    });
+}',
+                '<?php
+if (1) {
+  $message = (new class() extends Foo
+  {
+    public function bar() { echo 1; }
+  });
+}',
+                self::$configurationOopPositionSameLine + self::$configurationAnonymousPositionNextLine,
+            ],
+            [
+                '<?php
+if (1)
+{
+    $message = (new class() extends Foo
+    {
+        public function bar()
+        {
+            echo 1;
+        }
+    });
+}',
+                '<?php
+if (1) {
+  $message = (new class() extends Foo
+  {
+    public function bar() { echo 1; }
+  });
+}',
+                self::$configurationCtrlStructPositionNextLine + self::$configurationAnonymousPositionNextLine,
+            ],
+            [
+                '<?php
+if (1)
+{
+    $message = (new class() extends Foo
+    {
+        public function bar() {
+            echo 1;
+        }
+    });
+}',
+                '<?php
+if (1) {
+  $message = (new class() extends Foo
+  {
+    public function bar() { echo 1; }
+  });
+}',
+                self::$configurationOopPositionSameLine + self::$configurationCtrlStructPositionNextLine + self::$configurationAnonymousPositionNextLine,
```

Comment:
This test encompasses the effect of all the brace moving options being applied, for an `inverted` look on what you'd expect from a PSR2 document.